### ISSUE TITLE
Question : Importing content from another system

### ISSUE THREAD

**Issue by ghassen1khalil** [drodil/backstage-plugin-qeta#126]
Is there a way to make this plugin fetch data (question) from another source (eg: stackoverflow) through API calls to its backend ?

**Comment by drodil** [comment]
No, there's no such support. Config allows defining `allowMetadataInput` which basically allows to post questions imported from another source with some script.

**Comment by ghassen1khalil** [comment]
> No, there's no such support. Config allows defining `allowMetadataInput` which basically allows to post questions imported from another source with some script.

Can you please tell me if there is any example to do so ?

**Comment by drodil** [comment]
@bkoski do you happen to have any example? Maybe something could be added to this repo as well

**Comment by drodil** [comment]
See also: https://github.com/drodil/backstage-plugin-qeta/discussions/147

**Comment by kirbarn** [comment]
I added some info on how we successfully migrated from Stack in #147

**Comment by drodil** [comment]
I think this can be closed as there is a discussion in #147
